OpenRA.Game/GameInformation.cs

```diff
@@ -143,6 +147,19 @@ public Player GetPlayer(OpenRA.Player runtimePlayer)
 			return player;
 		}
 
+		public string ResolvedPlayerName(Player player)
+		{
+			if (player.IsBot)
+			{
+				var botsOfSameType = Players.Where(p => p.BotType == player.BotType).ToArray();
+				return TranslationProvider.GetString(EnumeratedBotName,
+					Translation.Arguments("name", TranslationProvider.GetString(player.Name),
+						"number", botsOfSameType.IndexOf(player) + 1));
```

Comment:
```suggestion
				var number = Players.Where(p => p.BotType == player.BotType).ToList().IndexOf(player) + 1;
				return TranslationProvider.GetString(EnumeratedBotName,
					Translation.Arguments(
						"name", TranslationProvider.GetString(player.Name),
						"number", number));
			}
```

I think this would be much more readable formatting